tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -225,7 +225,7 @@ jobs:
         inputs:
           command: 'custom'
           workingDir: ${{ parameters.testWorkspace }}
-          customCommand: 'install $(Initialize.testPackageTgz) ${{ parameters.loggerPackage }}'
+          customCommand: 'install $(Initialize.testPackageTgz)' # ${{ parameters.loggerPackage }}'
```

Comment:
Why?

Reply:
I thought this was supposed to post logs to Kusto?